test/controllers/application_controller_test.rb

```diff
@@ -1,4 +1,4 @@
-require 'test helper'
+require 'test_helper'
```

Comment:
Ah never mind, the previous comment ;)
This would be a good commit to `amend` with the previous one. I can show you how tomorrow and we can talk about when it is good to change pushed code and when it isn't a good idea.